--- icu/source/icudefs.mk.in.orig	2011-04-07 17:15:44.000000000 +0200
+++ icu/source/icudefs.mk.in	2011-04-07 17:15:56.000000000 +0200
@@ -189,14 +189,13 @@
 
 ## If we can't use the shared libraries, use the static libraries
 ifneq ($(ENABLE_SHARED),YES)
-STATIC_PREFIX_WHEN_USED = s
+STATIC_PREFIX_WHEN_USED =
 else
 STATIC_PREFIX_WHEN_USED = 
 endif
 
 # Static library prefix and file extension
-STATIC_PREFIX = s
-LIBSICU = $(LIBPREFIX)$(STATIC_PREFIX)$(ICUPREFIX)
+LIBSICU = $(LIBPREFIX)$(ICUPREFIX)
 A = a
 SOBJ = $(SO)
 
